.chronus/changes/auto-swathipil-http-client-python-add-pyproject-toml-2025-6-21-14-58-56.md

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
TODO: Also check multiapi config in autorest.python for swagger codegen. If so, set basic-setup-py to true. This way, if multiapi SDKs need regeneration, basic-setup-py won't need to be added to all of them.